### PR TITLE
Subgraph package app router

### PR DESCRIPTION
**Pull request by kmjones1979**
## Description

Merging into original subgraph-package name

## Additional Information

- [X] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [X] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

NA

Your ENS/address: Shutterblock.eth

### CODE REVIEW

File: packages/nextjs/package.json [hunk 1]
```diff
@@ -14,20 +14,24 @@
     "vercel:yolo": "vercel --build-env NEXT_PUBLIC_IGNORE_BUILD_ERROR=true"
   },
   "dependencies": {
+    "@apollo/client": "^3.9.4",
     "@ethersproject/providers": "^5.7.2",
     "@heroicons/react": "^2.0.11",
     "@rainbow-me/rainbowkit": "1.3.0",
     "@uniswap/sdk-core": "^4.0.1",
     "@uniswap/v2-sdk": "^3.0.1",
+    "@urql/core": "^4.2.3",
```
Comment (schmidsi) [inline packages/nextjs/package.json:23]: It seems like URQL is not used, so why install it?

File: packages/subgraph/build/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+type Greeting @entity {
```
Comment (schmidsi) [inline packages/subgraph/build/schema.graphql:1]: `build` and `generated` folders are usually ignored. Any reason to commit it here?

File: packages/nextjs/package.json (hunk 1 above)
Comment (kmjones1979) [reply packages/nextjs/package.json:23]: Thanks! I missed that - cleaned this up here https://github.com/scaffold-eth/scaffold-eth-2/pull/722

File: packages/subgraph/build/schema.graphql (hunk 2 above)
Comment (kmjones1979) [reply packages/subgraph/build/schema.graphql:1]: also cleaned up here: https://github.com/scaffold-eth/scaffold-eth-2/pull/722

**Pull request merged**